sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/api/Microsoft.Azure.WebJobs.Extensions.ServiceBus.netstandard2.0.cs

```diff
@@ -65,6 +65,15 @@ public partial class MessagingProvider
         public virtual Azure.Messaging.ServiceBus.ServiceBusProcessor CreateProcessor(string entityPath, string connectionString) { throw null; }
         public virtual Microsoft.Azure.WebJobs.ServiceBus.SessionMessageProcessor CreateSessionMessageProcessor(string entityPath, string connectionString) { throw null; }
     }
+    public partial class ServiceBusMessageActions
+    {
+        internal ServiceBusMessageActions() { }
```

Comment:
Should add protected ctor for mocking (or make ctors public).

Reply:
Make it abstract?